Multisig/Data/Services/SafeTransactionService.swift

```diff
@@ -45,3 +49,29 @@ struct SafeStatusRequest: JSONRequest {
         let version: String
     }
 }
+
+struct SafeBalancesRequest: JSONRequest {
+    let address: String
```

Comment:
I would keep this as `var`, because the whole struct can be made immutable with `let`

Reply:
What is an additional value in it? If you are not going to change smth, why not make it 'let'. If the case like this appears, I would rather make it 'var' later.

Reply:
the meaning of the `let` is that it declares a constant value that is immutable. In Swift, structs themselves are immutable. 

When I read `let` inside of the struct, I read it like "here is the constant address". And I wonder why the address cannot change? 

The `var` is the default choice, I would keep it as it is, having the `let` here constrains this struct from other usage and disallows me to change just one value of the struct - I need to re-initialize this struct all together with all the values the same except that one.

I think this does not bring much value, and it confuses me when I read it.

Reply:
surely I should be able to construct a request with one address, and if I want, change that address property to another value. But I won't be able to do that, because the address is a constant. 

And then I become frustrated.

Reply:
I don't want to come back to this code later, this makes this code not reusable, as it would be reusable by default

Reply:
in general, one of the design objective is to anticipate the future changes

Reply:
discussed. Decision: not to change in this PR.